google/resource_bigquery_dataset_test.go

```diff
@@ -112,3 +159,97 @@ resource "google_bigquery_dataset" "test" {
   }
 }`, datasetID)
 }
+
+func testAccBigQueryDatasetWithOneAccess(datasetID string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "access_test" {
+  dataset_id                  = "%s"
+
+  access {
+    role          = "OWNER"
+    user_by_email = "Joe@example.com"
+  }
+
+  labels {
+    env                         = "foo"
+    default_table_expiration_ms = 3600000
+  }
+}`, datasetID)
+}
+
+func testAccBigQueryDatasetWithTwoAccess(datasetID string) string {
+	return fmt.Sprintf(`
+resource "google_bigquery_dataset" "access_test" {
+  dataset_id                  = "%s"
+
+  access {
+    role          = "OWNER"
+    user_by_email = "Joe@example.com"
+  }
+  access {
+    role	      = "READER"
+    domain	      = "example.com"
+  }
+
+  labels {
+    env                         = "foo"
+    default_table_expiration_ms = 3600000
+  }
+}`, datasetID)
+}
+
+func getBigQueryTableWithView(datasetID, tableID string) string {
```

Comment:
Is this func only used in the one place? If so, may as well just inline it

Reply:
Done.